index.html

```diff
@@ -3856,29 +3907,31 @@ <h3 id="Warrior">Warrior</h3>
             <tr>
                <td>8
                </td>
-               <td>Silencing Strike
+               <td>Thunderous Slam
                </td>
-               <td><strong>Lash out and limit an enemy’s capabilities.</strong>
+               <td><strong>Summon an overwhelming sonic boom to bring your enemies to their
+                     knees. </strong>
                   <p>
-                     Make a melee weapon attack against a creature and on a hit the creature is
-                     <em>silenced</em>. 2 FORT points to resist the <em>silence</em>.
+                     Make a melee weapon attack against a creature. On a hit all creatures within
```

Comment:
Would you be happy with half that damage? Or are we happy with it just reducing AC? I felt it may be a little boring with just the AC reduction as AC will generally be low anyway but if people will still pick it I'm happy.